connectors/huggingface_connector.py

```diff
@@ -0,0 +1,90 @@
+from datasets import load_dataset, get_dataset_split_names
+from nomic import AtlasDataset
+import pyarrow as pa
+import pyarrow.compute as pc
+
+
+# Gets data from HF dataset
+def get_hfdata(dataset_identifier, split="train", limit=100000):
+    try:
+        # Determine available splits
+        splits = get_dataset_split_names(dataset_identifier)
+        if split not in splits:
+            # Use the first available split if the specified split is not found
+            split = splits[0]
+
+
+        # Load the dataset
+        dataset = load_dataset(dataset_identifier, split=split, streaming=True)
+    except ValueError as e:
```

Comment:
We don't need to handle this error, seems like handling config through the error message is too risky

Reply:
So you can take out the try/except here